File: python/challenge-2025-09-30.py

def format_number(number):
    country_code    = number[:1]
    area_code       = number[1:4]
    part_1          = number[4:7]
    part_2          = number[7:]
    result = f"+{country_code} ({area_code}) {part_1}-{part_2}"
    return result

File: javascript/challenge-2025-09-30.js

function formatNumber(number) {
  const country_code  = number.slice(0, 1);
  const area_code     = number.slice(1, 4);
  const part_1        = number.slice(4, 7);
  const part_2        = number.slice(7);
  const result = `+${country_code} (${area_code}) ${part_1}-${part_2}`;
  return result;
}



